lib/pg_ha_migrations/safe_statements.rb

```diff
@@ -286,6 +300,76 @@ def safe_create_partitioned_table(table, key:, type:, infer_primary_key: nil, **
     end
   end
 
+  def safe_partman_create_parent(table, key:, interval:, **options)
```

Comment:
Should this have explicit keyword args instead of options? Then we get the validation of valid option names for free.

Reply:
That is true, but it comes at the cost of a really long method signature and would require us to manually build the hash of arguments here: https://github.com/braintree/pg_ha_migrations/blob/a0275cda749115a94122a0e42bed5c915e105cdf/lib/pg_ha_migrations/safe_statements.rb#L317-L322

Reply:
Yeah, I'm not saying there are no downsides, but I think it also offers real benefit to the callers of the code, because there's self-documentation in the signature.

Reply:
Because this method is also passing along args to `unsafe_partman_update_config`, I can sort of make the same argument that I made below. However, the only column that is actually nullable is `retention` and that value is null by default, so there's no difference between calling this method with `retention: nil` and just omitting the argument altogether. So that's my long winded way of saying I implemented your proposed change and pushed up a commit 😆